ArjunaJTS/orbportability/classes/com/arjuna/orbportability/ORBType.java

```diff
@@ -67,16 +67,14 @@ static int getORBEnum(String name)
     public static final int HPORB = 2;
     public static final int VISIBROKER = 3;
     public static final int JAVAIDL = 1;
-	public static final int JACORB = 4;
     public static final int IBMORB = 5;
 
     public static final String orbix2000 = "ORBIX2000";
     public static final String hporb = "HPORB";
     public static final String visibroker = "VISIBROKER";
     public static final String javaidl = "JAVAIDL";
-	public static final String jacorb = "JACORB";
 	public static final String ibmorb = "IBMORB";
 
-    private final static String[]           ORB_NAME = { orbix2000, hporb, visibroker, javaidl, ibmorb, jacorb };
-    private final static int[]              ORB_ENUM = { ORBIX2000, HPORB, VISIBROKER, JAVAIDL, IBMORB, JACORB };
+    private final static String[]           ORB_NAME = { orbix2000, hporb, visibroker, javaidl, ibmorb };
```

Comment:
Follow up work will presumably be to remove orbix2000, hporb, visibroker from this list? Would you be able to raise a JBTM for that, please?

Reply:
Agreed. [JBTM-3635](https://issues.redhat.com/browse/JBTM-3635)